Engine/game/savegame_internal.h

```diff
@@ -86,7 +86,7 @@ struct RestoredData
         int PanAsPercent;
         int Speed;
     };
-    ChannelInfo             AudioChans[MAX_SOUND_CHANNELS + 1];
+    ChannelInfo             AudioChans[MAX_SOUND_CHANNELS];
     // Ambient sounds
```

Comment:
Same as in my comment above, this should remain [MAX_SOUND_CHANNELS + 1 because save keeps state of extra 1 crossfade channel too.

Reply:
This was an accident. I didn't mean to change any array lengths in this commit. Ignore this, I will revert it